google-cloud-spanner/acceptance/spanner/client/params/date_test.rb

```diff
@@ -15,56 +15,86 @@
 require "spanner_helper"
 
 describe "Spanner Client", :params, :date, :spanner do
-  let(:db) { spanner_client }
+  let :db do
+    { gsql: spanner_client, pg: spanner_pg_client }
+  end
+  let :date_query do
+    { gsql: "SELECT @value AS value",
+      pg: "SELECT $1 AS value" }
+  end
   let(:date_value) { Date.today }
 
-  it "queries and returns a date parameter" do
-    results = db.execute_query "SELECT @value AS value", params: { value: date_value }
+  dialects = [:gsql]
+  dialects.push :pg unless emulator_enabled?
 
-    _(results).must_be_kind_of Google::Cloud::Spanner::Results
-    _(results.fields[:value]).must_equal :DATE
-    _(results.rows.first[:value]).must_equal date_value
-  end
+  dialects.each do |dialect|
+    it "queries and returns a date parameter for #{dialect}" do
+      results = db[dialect].execute_query date_query[dialect],
+                                          params: dialect === :gsql ? { value: date_value } : { p1: date_value }
```

Comment:
Please avoid using `===` unless absolutely necessary. (Remember `===` is very different in Ruby than in Javascript.)